### PR TITLE
Refactor README on different docker version.

### PR DESCRIPTION
**Pull request by JinHai-CN**
### What problem does this PR solve?

1. Use two env files for slim and full docker image.
2. Update README

### Type of change

- [x] Documentation Update
- [x] Refactoring

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -138,12 +138,18 @@
    ```bash
    $ cd ragflow/docker
    $ chmod +x ./entrypoint.sh
-   $ docker compose -f docker-compose.yml up -d
+   $ docker compose --env-file slim.env up -d
    ```
 
-   > 请注意，运行上述命令会自动下载 RAGFlow 的开发版本 docker 镜像。如果你想下载并运行特定版本的 docker 镜像，请在 docker/.env 文件中找到 RAGFLOW_IMAGE 变量，将其改为对应版本。例如 `RAGFLOW_IMAGE=infiniflow/ragflow:v0.12.0-slim`，然后运行上述命令。
+   > 运行上述命令会自动下载 RAGFlow 的 slim-dev 版本的 docker 镜像，该镜像并不包含 embedding 模型以及很多 Python 库，因此镜像大小约 1GB。如果你想下载并运行特定版本的 docker 镜像，请在 docker/slim.env 文件中找到 RAGFLOW_IMAGE 变量，将其改为对应版本。例如 `RAGFLOW_IMAGE=infiniflow/ragflow:v0.12.0-slim`，然后运行上述命令。
```
Comment (writinwaters) [inline README_zh.md]:
```suggestion
   > 运行上述命令会自动下载 RAGFlow 的 slim-dev 版本的 Docker 镜像，该镜像并不包含 embedding 模型以及很多 Python 库，因此镜像大小约 1GB。如果你想下载并运行特定版本的 Docker 镜像，请在 docker/slim.env 文件中找到 RAGFLOW_IMAGE 变量，将其改为对应版本。例如 `RAGFLOW_IMAGE=infiniflow/ragflow:v0.12.0-slim`，然后再运行上述命令。
```

File: docker/slim.env [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+MINIO_PASSWORD=infini_rag_flow
+
+REDIS_PORT=6379
+REDIS_PASSWORD=infini_rag_flow
+
+SVR_HTTP_PORT=9380
+
+RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim
+
+# If can't download RAGFlow docker image, try uncomment either of the following hub.docker.com mirrors:
```
Comment (writinwaters) [inline docker/slim.env]:
```suggestion
# If you cannot download the RAGFlow Docker image, try uncommenting either of the following hub.docker.com mirrors:
```

File: docker/slim.env [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# RAGFLOW_IMAGE=swr.cn-north-4.myhuaweicloud.com/infiniflow/ragflow:dev-slim
+# RAGFLOW_IMAGE=registry.cn-hangzhou.aliyuncs.com/infiniflow/ragflow:dev-slim
+
+TIMEZONE='Asia/Shanghai'
+
+# If can't download RAGFlow docker image, try to uncomment the following huggingface.co mirror:
```
Comment (writinwaters) [inline docker/slim.env]:
```suggestion
# If you cannot download the RAGFlow docker image, try uncommenting the following huggingface.co mirror:
```

File: docker/slim.env [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+# HF_ENDPOINT=https://hf-mirror.com
+
+######## OS setup for ES ###########
+# sysctl vm.max_map_count
+# sudo sysctl -w vm.max_map_count=262144
+# However, this change is not persistent and will be reset after a system reboot.
```
Comment (writinwaters) [inline docker/slim.env]:
```suggestion
# Note that this change is not permanent and will be reset after a system reboot.
```

File: docker/slim.env [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+# To make the change permanent, you need to update the /etc/sysctl.conf file.
```
Comment (writinwaters) [inline docker/slim.env]:
```suggestion
# To make your change permanent, update /etc/sysctl.conf by:
```

File: docker/slim.env [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+# Add or update the following line in the file:
```
Comment (writinwaters) [inline docker/slim.env]:
```suggestion
# Adding or modifying the following line:
```

File: docker/slim.env [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+# Set the Elasticsearch password
+ELASTIC_PASSWORD=infini_rag_flow
+
+# Port to expose Kibana to the host
+KIBANA_PORT=6601
+KIBANA_USER=rag_flow
+KIBANA_PASSWORD=infini_rag_flow
+
+# Increase or decrease based on the available host memory (in bytes)
+
```
Comment (writinwaters) [inline docker/slim.env]: Delete this additional line.

File: docker/slim.env [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+# Set the Elasticsearch password
+ELASTIC_PASSWORD=infini_rag_flow
+
+# Port to expose Kibana to the host
+KIBANA_PORT=6601
+KIBANA_USER=rag_flow
+KIBANA_PASSWORD=infini_rag_flow
+
+# Increase or decrease based on the available host memory (in bytes)
```
Comment (writinwaters) [inline docker/slim.env]:
```suggestion
# Update according to the available host memory (in bytes)
```

**Review by writinwaters — CHANGES_REQUESTED** [review]

**Review by writinwaters — APPROVED** [review]

File: README_zh.md [hunk 9]
```diff
@@ -140,9 +140,15 @@
    $ docker compose -f docker-compose.yml up -d
    ```
 
-   > 请注意，运行上述命令会自动下载 RAGFlow 的开发版本 docker 镜像。如果你想下载并运行特定版本的 docker 镜像，请在 docker/.env 文件中找到 RAGFLOW_IMAGE 变量，将其改为对应版本。例如 `RAGFLOW_IMAGE=infiniflow/ragflow:v0.12.0-slim`，然后运行上述命令。
+   > 运行上述命令会自动下载 RAGFlow 的 slim-dev 版本的 Docker 镜像，该镜像并不包含 embedding 模型以及很多 Python 库，因此镜像大小约 1GB。如果你想下载并运行特定版本的 Docker 镜像，请在 docker/slim.env 文件中找到 RAGFLOW_IMAGE 变量，将其改为对应版本。例如 `RAGFLOW_IMAGE=infiniflow/ragflow:v0.12.0-slim`，然后再运行上述命令。
```
Comment (yuzhichang) [inline README_zh.md]: `slim-dev`  -> `dev-slim`

File: docker/.env [hunk 10]
```diff
@@ -33,21 +31,39 @@ REDIS_PASSWORD=infini_rag_flow
 
 SVR_HTTP_PORT=9380
 
-RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim
+RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim # slim version docker image
```
Comment (writinwaters) [inline docker/.env]: RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim # the Docker image for the slim version

File: docker/.env [hunk 11]
```diff
@@ -33,21 +31,39 @@ REDIS_PASSWORD=infini_rag_flow
...
 
-# If can't download RAGFlow docker image, try to uncomment either of the following hub.docker.com mirrors:
+# If you cannot download the RAGFlow Docker image, try uncommenting either of the following hub.docker.com mirrors:
 # RAGFLOW_IMAGE=swr.cn-north-4.myhuaweicloud.com/infiniflow/ragflow:dev-slim
 # RAGFLOW_IMAGE=registry.cn-hangzhou.aliyuncs.com/infiniflow/ragflow:dev-slim
 
+# If you want to download the RAGFlow docker image with embedding models. You may modify above as following:
```
Comment (writinwaters) [inline docker/.env]:
```suggestion
# To download the RAGFlow docker image including embedding models, modify the line above as follows:
```

File: docker/.env [hunk 12]
```diff
@@ -33,21 +31,39 @@ REDIS_PASSWORD=infini_rag_flow
...
+# RAGFLOW_IMAGE=infiniflow/ragflow:dev # non-slim version docker image
+
+# In this docker image, following two models are included.
```
Comment (writinwaters) [inline docker/.env]:
```suggestion
# This Docker image includes the following two models:
```

**Review by yuzhichang — APPROVED** [review]

**Review by writinwaters — CHANGES_REQUESTED** [review]

File: docker/.env [hunk 13]
```diff
@@ -33,21 +31,45 @@ REDIS_PASSWORD=infini_rag_flow
 
 SVR_HTTP_PORT=9380
 
+# slim version Docker image
```
Comment (writinwaters) [inline docker/.env]:
```suggestion
# Docker image for the slim version
```

File: docker/.env [hunk 14]
```diff
@@ -33,21 +31,45 @@ REDIS_PASSWORD=infini_rag_flow
...
 RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim
 
-# If can't download RAGFlow docker image, try to uncomment either of the following hub.docker.com mirrors:
+# If you cannot download the RAGFlow Docker image, try uncommenting either of the following hub.docker.com mirrors:
 # RAGFLOW_IMAGE=swr.cn-north-4.myhuaweicloud.com/infiniflow/ragflow:dev-slim
 # RAGFLOW_IMAGE=registry.cn-hangzhou.aliyuncs.com/infiniflow/ragflow:dev-slim
 
+# If you want to download the RAGFlow Docker image with embedding models. You may modify above as following:
```
Comment (writinwaters) [inline docker/.env]:
```suggestion
# To download the RAGFlow Docker image with embedding models, modify the line above as follows:
```

File: docker/.env [hunk 15]
```diff
@@ -33,21 +31,45 @@ REDIS_PASSWORD=infini_rag_flow
...
+# RAGFLOW_IMAGE=infiniflow/ragflow:dev
+
+# In this Docker image, following 4 models are included.
```
Comment (writinwaters) [inline docker/.env]:
```suggestion
# This Docker image includes the following four models:
```
Spell out any digit below ten.

File: docker/.env [hunk 16]
```diff
@@ -33,21 +31,45 @@ REDIS_PASSWORD=infini_rag_flow
...
+# - BAAI/bge-large-zh-v1.5
+# - BAAI/bge-reranker-v2-m3
+# - maidalun1020/bce-embedding-base_v1
+# - maidalun1020/bce-reranker-base_v1
+
+# And following models will be downloaded when user choose it in RAGFlow UI.
```
Comment (writinwaters) [inline docker/.env]:
```suggestion
# And the following models will be downloaded if you select them in the RAGFlow UI.
```

**Review by writinwaters — CHANGES_REQUESTED** [review]

**Review by writinwaters — APPROVED** [review]

**Pull request merged**